src/List/List.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const List = ({ items, clickHandler, className, field = 'name' }) => (
+  <ul className={ className || null }>{ items.map( item => 
+    <li 
+      key={ item.id || Date.now() + Math.random() } 
```

Comment:
it is not good idea to use random in this place (in render method as well). If you don't have id it's better to use index